experimental/packages/otlp-exporter-base/package.json

```diff
@@ -64,12 +64,10 @@
     "@types/semver": "7.3.9",
```

Comment:
I guess this can be also removed

Reply:
Done in [d138210](https://github.com/open-telemetry/opentelemetry-js/pull/2893/commits/d13821077d6c26fff43fc29d3aa9a5591460b156). Thanks!